helfomat-infrastructure-thwde/src/test/kotlin/de/helfenkannjeder/helfomat/infrastructure/thwde/EmbeddedHttpServer.kt

```diff
@@ -0,0 +1,60 @@
+package de.helfenkannjeder.helfomat.infrastructure.thwde
+
+import com.sun.net.httpserver.HttpExchange
+import com.sun.net.httpserver.HttpHandler
+import com.sun.net.httpserver.HttpServer
+import org.springframework.core.io.ClassPathResource
+import org.springframework.util.FileCopyUtils
+import org.springframework.util.SocketUtils
+import java.net.InetSocketAddress
+import java.util.*
+
+object EmbeddedHttpServer {
```

Comment:
Are you sure this should be an object? Better make it a singleton and inject it where you need it.